src/tiled/commanddatamodel.cpp

```diff
@@ -433,6 +434,22 @@ bool CommandDataModel::dropMimeData(const QMimeData *data, Qt::DropAction, int,
     return false;
 }
 
+QKeySequence CommandDataModel::shortcut(const QModelIndex &index)
+{
+    const bool isNormalRow = index.row() < mCommands.size();
+
+    if (isNormalRow) {
+        return mCommands[index.row()].shortcut;
+    }
+    else
+        return QKeySequence();
```

Comment:
I'd prefer a rather shorter implementation:
```c++
if (index.isValid())
    return mCommands[index.row()].shortcut;
return QKeySequence();
```
Also this method should be declared `const`.